ingestion/src/metadata/orm_profiler/profiler/core.py

```diff
@@ -438,9 +474,19 @@ def get_profile(self) -> CreateTableProfileRequest:
             # computing metrics, if the type is not supported.
             # Let's filter those out.
             computed_profiles = [
-                ColumnProfile(**self.column_results.get(col.name))
+                ColumnProfile(
+                    **self.column_results.get(
+                        col.name
+                        if not isinstance(col.name, ColumnName)
+                        else col.name.__root__
+                    )
+                )
                 for col in self.columns
-                if self.column_results.get(col.name)
+                if self.column_results.get(
+                    col.name
+                    if not isinstance(col.name, ColumnName)
+                    else col.name.__root__
+                )
```

Comment:
Same than above we should not do all these if not isinstance() but try to make it so that it supports it.

Reply:
resolved

Reply:
Looks like we are still doing that check on the SQA ColumnName object type